openid/store/nonce.py

```diff
@@ -84,6 +85,19 @@ def checkTimestamp(nonce_string, allowed_skew=SKEW, now=None):
         return past <= stamp <= future
 
 
+def make_nonce_salt(length=6):
+    """
+    Generate and return a nonce salt.
+
+    @param length: Length of the generated string.
+    @type length: int
+    @rtype: six.text_type
+    """
+    sys_random = random.SystemRandom()
```

Comment:
This can raise `NotImplementedError` on some platforms.

Reply:
Yes, it can. But I consider it as the best solution. It seems better than using not-very-random source.

Reply:
I think that adding a fallback is a reasonable choice :) Also `secrets` module on p3.6 provides a nice interface to cryptography

Reply:
`secrets` is just wrap over `SystemRandom`, so it doesn't offer any additional fallback.

I'd rather have error raised, than fallback to not very secure Mersenne Twister.